pkgs/applications/window-managers/i3/default.nix

```diff
@@ -79,16 +139,20 @@ stdenv.mkDerivation rec {
 
   separateDebugInfo = true;
 
-  passthru.tests = { inherit (nixosTests) i3wm; };
-
+  passthru.tests = {
+    inherit (nixosTests) i3wm;
+  };
 
   meta = with lib; {
     description = "Tiling window manager";
-    homepage    = "https://i3wm.org";
-    maintainers = with maintainers; [ modulistic fpletz ];
+    homepage = "https://i3wm.org";
+    maintainers = with maintainers; [
+      modulistic
+      fpletz
+    ];
     mainProgram = "i3";
-    license     = licenses.bsd3;
-    platforms   = platforms.all;
+    license = licenses.bsd3;
+    platforms = platforms.all;
```

Comment:
all? I wonder if it works on darwin.

Reply:
Apparently, it does work on darwin: https://www.reddit.com/r/unixporn/comments/1bly4i3/i3_i3_on_macos/ :joy: 

I'm going to restrict the platforms to unix, though, IMHO that makes the most sense.